cmd/nameserver/config.example.yaml

```diff
@@ -0,0 +1,31 @@
+server:
+	serviceapi: 0.0.0.0:8000
```

Comment:
Please, use 2 space indentation like in the rest of the file.

Reply:
done